### ISSUE TITLE
iOS10.3.3 offline-capable form not loading

### ISSUE THREAD

**Issue by MartijnR** [kobotoolbox/enketo-express#1125]
stuck on loading screen.

There is an ee.kobotoolbox.org user on iOS10 that can no longer load a form. That iOS version is not supported any more, but the problem is that he has records in the queue.

**Comment by MartijnR** [comment]
https://groups.google.com/forum/#!topic/enketo-users/ZQCeR0_cC48

**Comment by MartijnR** [comment]
The issue is that this browser loads the IE11 polyfills (because it doesn't support`module`)

**Comment by MartijnR** [comment]
<img width="1193" alt="screen shot 2018-10-22 at 3 11 06 pm" src="https://user-images.githubusercontent.com/627350/47319757-c81b8b00-d60c-11e8-9bc4-37b12238f305.png">

**Comment by MartijnR** [comment]
iOS11 also loads the polyfill, but this does not cause an exception.

**Comment by MartijnR** [comment]
Final report:

Some weeks back, I've moved all IE11 polyfills to a separate file using the wonderful `nomodule` trick. This avoids modern browsers from loading all that shit (I love this!). Unfortunately the very dated iOS10 browsers load it too, and there is some issue in at least one of the IE11 polyfills in iOS10. That would be an enormous amount of work to track down for 1 user, and we'll likely find more issues after fixing the first.

The most attractive way to enable him to submit his records would be to disable IE11 support (otherwise, you'd have to temporarily downgrade and losing features). I have created a `fix/disable-ie11 branch` that does this dirtily.  I'm wondering though whether KoBo is ready to officially drop IE11 support and redirect to https://ee.kobotoolbox.org/modern-browsers. (Note we had dropped IE11 support years ago but brought it back for OpenClinica). In that case, I'll add the redirect and create a config.json item to disable IE11 support properly.

**Comment by tylerhoecker** [comment]
Thank you Martijn for tracking down this issue. I apologize to everybody for making this request. I realize my OS was out of date, and you cannot continue to support old systems. 

I would be very grateful if we could temporarily make these changes so that I can upload my data. We collected these data in a remote field location that we cannot return to, and so it is important that I access the data if at all possible. Thanks for providing these services, which we have enjoyed using! 

I am happy to assist in any way that I can to make this possible and minimize disruption to other users and your time.
